<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBarCustomizer</class>
 <widget class="QDialog" name="ToolBarCustomizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customize tool-bar</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxAvailableActions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Available tool-bar actions</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutAvailableActions">
      <item row="0" column="0">
       <widget class="QListWidget" name="listWidgetAvailableActions">
        <property name="toolTip">
         <string>List of available tool-bar actions</string>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonActivateActions">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Activate selected tool-bar actions</string>
        </property>
        <property name="text">
         <string>Activate</string>
        </property>
        <property name="icon">
         <iconset resource="../qmc2.qrc">
          <normaloff>:/data/img/right_action.png</normaloff>:/data/img/right_action.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxActiveActions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Active tool-bar actions</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutActiveActions">
      <item row="0" column="0" colspan="5">
       <widget class="QListWidget" name="listWidgetActiveActions">
        <property name="toolTip">
         <string>List of active tool-bar actions and their order</string>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonDeactivateActions">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Deactivate selected tool-bar actions</string>
        </property>
        <property name="icon">
         <iconset resource="../qmc2.qrc">
          <normaloff>:/data/img/left_action.png</normaloff>:/data/img/left_action.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonActionUp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Move selected action up</string>
        </property>
        <property name="icon">
         <iconset resource="../qmc2.qrc">
          <normaloff>:/data/img/up_action.png</normaloff>:/data/img/up_action.png</iconset>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonActionDown">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Move selected action down</string>
        </property>
        <property name="icon">
         <iconset resource="../qmc2.qrc">
          <normaloff>:/data/img/down_action.png</normaloff>:/data/img/down_action.png</iconset>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButtonInsertSeparator">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Insert a separator line</string>
        </property>
        <property name="text">
         <string>Separator</string>
        </property>
        <property name="icon">
         <iconset resource="../qmc2.qrc">
          <normaloff>:/data/img/plus.png</normaloff>:/data/img/plus.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButtonDefault">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Reset tool-bar actions to default configuration</string>
        </property>
        <property name="text">
         <string>Default</string>
        </property>
        <property name="icon">
         <iconset resource="../qmc2.qrc">
          <normaloff>:/data/img/broom.png</normaloff>:/data/img/broom.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>208</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="toolTip">
        <string>Apply tool-bar setup and close dialog</string>
       </property>
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset resource="../qmc2.qrc">
         <normaloff>:/data/img/yes.png</normaloff>:/data/img/yes.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="toolTip">
        <string>Apply tool-bar setup</string>
       </property>
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../qmc2.qrc">
         <normaloff>:/data/img/filesave.png</normaloff>:/data/img/filesave.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="toolTip">
        <string>Cancel tool-bar setup and close dialog</string>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../qmc2.qrc">
         <normaloff>:/data/img/no.png</normaloff>:/data/img/no.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>208</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidgetAvailableActions</tabstop>
  <tabstop>listWidgetActiveActions</tabstop>
  <tabstop>pushButtonActivateActions</tabstop>
  <tabstop>pushButtonDeactivateActions</tabstop>
  <tabstop>pushButtonActionUp</tabstop>
  <tabstop>pushButtonActionDown</tabstop>
  <tabstop>pushButtonInsertSeparator</tabstop>
  <tabstop>pushButtonDefault</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonApply</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../qmc2.qrc"/>
 </resources>
 <connections/>
</ui>
